src/molecules/Dialog/Dialog.tsx

```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
       children,
       width,
       actions,
-      withContentPadding = true,
+      withContentPaddingX = true,
+      withContentPaddingY = true,
       withBorders = false,
       contentMaxHeight = '60vh',
+      hideClose,
```

Comment:
Couldn't hideClose instead be achieved with onClose being optional?

Reply:
`onClose` is also used by the cancel button, but i added a comment in the property description to specify this, hope that works?